_data/simple-icons.json

```diff
@@ -1587,6 +1587,11 @@
             "hex": "00665E",
             "source": "https://www.astonmartin.com"
         },
+        {
+            "title": "Astra",
+            "hex": "FFC300",
+            "source": "https://cdn.shortpixel.ai/spai/ret_img/wpastra.com/wp-content/uploads/2019/05/astra-logo.svg"
```

Comment:
See https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines, this is incorrect.

Reply:
Yeah right. Just updated the source.